sections/browsers.include

```diff
@@ -3517,22 +3517,22 @@
 
     <p class="note">
   The <i>handle redirects</i> step later in
-    this algorithm can in certain cases jump back to the step labeled <a><i>fragment identifiers</i></a>. Since, between those two steps,
+    this algorithm can in certain cases jump back to the step labeled <i>fragments</i>. Since, between those two steps,
```

Comment:
I make these `<var>`s and adjust case as necessary so that there's at least two instances so that Bikeshed doesn't complain.